app.js

```diff
@@ -0,0 +1,18 @@
+const express = require("express");
+const app = express();
+const { getTopics } = require("./controllers/topics-controllers");
+
+app.use(express.json());
+
+app.get("/api/topics", getTopics);
+
+app.use((req, res, next) => {
```

Comment:
This piece of error handling middleware isn't actually being used atm. Our bad URLs test is passing as express catches unhandled URLs and sends a 404